server/webhook_parser.go

```diff
@@ -310,7 +309,11 @@ func appendCommentNotifications(wh *webhook, verb string) {
 }
 
 func quoteIssueComment(comment string) string {
-	return fmt.Sprintf("> %s", comment)
+	lines := strings.Split(comment, "\n")
+	for i, line := range lines {
+		lines[i] = "> " + line
+	}
+	return strings.Join(lines, "\n")
```

Comment:
This function can actually be shortened to one line like so:

```suggestion
	return "> " + strings.ReplaceAll(comment, "\n", "\n> ")
```

I recommend making this last change before merging